scripts/start_container.sh

```diff
@@ -3,10 +3,16 @@
 PORT=${PORT:-8888}
```

Comment:
it wouldn't be so hard to figure out that open port, but more than this use case is worth, I think!  If the above bug is fixed, this script supports, for example `PORT=8889 ./scripts/start_container.sh` to start a notebook running on an alternate port.